docs/content/3.forms/1.input.md

```diff
@@ -159,6 +159,21 @@ excludedProps:
 ---
 ::
 
+### Type
+
+Use the `type` prop to change the input type, the default `type` is set to `text`, you can check all the available types at [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types).
+
+We have improved the implementation of certain types such as [Checkbox](/forms/checkbox), [Radio](/forms/radio), etc.
+
+::component-card
+---
+baseProps:
+  name: 'input'
+props:
+  type: 'password'
+---
+::
+
```

Comment:
Nice, I think it's good to add this after style or size.

Reply:
Sure, moved it.